server/tests/hybrid/snapshots/snap_test_hybrid.py

```diff
@@ -44,9 +44,9 @@
 J2,Audit Board #1,,,,\r
 \r
 ######## ROUNDS ########\r
-Round Number,Contest Name,Targeted?,Sample Size,Risk Limit Met?,P-Value,Start Time,End Time,Audited Votes\r
-1,Contest 1,Targeted,22,No,0.7747175905,DATETIME,DATETIME,Choice 1-1: 7; Choice 1-2: 1\r
-1,Contest 2,Opportunistic,,No,1.0,DATETIME,DATETIME,Choice 2-1: 4; Choice 2-2: 1; Choice 2-3: 0\r
+Round Number,Contest Name,Targeted?,Sample Size,Risk Limit Met?,P-Value,Start Time,End Time,Audited Votes,Audited Votes: CVR,Audited Votes: Non CVR\r
+1,Contest 1,Targeted,22,No,0.7747175905,DATETIME,DATETIME,Choice 1-1: 13; Choice 1-2: 6,Choice 1-2: 5; Choice 1-1: 6,Choice 1-1: 7; Choice 1-2: 1\r
```

Comment:
looks like `choice 1-2` comes before `choice 1-1` in the `Audited Votes: CVR` field value. Should that be standardized to the same order?

Reply:
good catch, fixed